src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryAppDeployer.java

```diff
@@ -108,14 +122,15 @@ public String deploy(AppDeploymentRequest request) {
 					.build())
 				.doOnSuccess(v -> logger.info(String.format("Done uploading bits for %s", name)))
 				.doOnError(e -> logger.error(String.format("Error creating app %s", name), e))
-				.after(() -> operations.applications().setEnvironmentVariable(
-					SetEnvironmentVariableApplicationRequest.builder()
-						.name(name)
-						.variableName("SPRING_APPLICATION_JSON")
-						.variableValue(argsAsJson)
-						.build())
-					.doOnSuccess(v -> logger.debug(String.format("Setting env for app %s as %s", name, argsAsJson)))
-					.doOnError(e -> logger.error(String.format("Error setting environment for app %s", name), e))
+				// TODO: Replace the following clause with an -operations API call
```

Comment:
TODO should have a corresponding github issue.  Will create one and edit TODO comment on merge.